CONTRIBUTING.rst

```diff
@@ -0,0 +1,163 @@
+Contributing
+============
+
+Contributions and issues are most welcome! All issues and pull requests are
+handled through GitHub_. Also, please check for any existing issues before
+filing a new one. If you have a great idea but it involves big changes, please
+file a ticket before making a pull request! We want to make sure you don't spend
+your time coding something that might not fit the scope of the project.
+
+.. _GitHub: https://github.com/epics-containers/python3-pip-skeleton/issues
+
+Running the tests
+-----------------
+
+To run in a container
+~~~~~~~~~~~~~~~~~~~~~
+
+Use vscode devcontainer as follows::
+
+    $ git clone git://github.com/epics-containers/python3-pip-skeleton.git
+    $ vscode python3-pip-skeleton
+    Click on 'Reopen in Container' when prompted
+    In a vscode Terminal:
+    $ tox -p
+
+
+To run locally
+~~~~~~~~~~~~~~
+
+Get the source source code and run the unit tests directly
+on your workstation as follows::
+
+    $ git clone git://github.com/epics-containers/python3-pip-skeleton.git
+    $ cd python3-pip-skeleton
+    $ virtualenv .venv
+    $ source .venv/bin/activate
+    $ pip install -e .[dev]
+    $ tox -p 
+
+In both cases tox -p runs in parallel the following checks:
+
+  - Build Sphinx Documentation
+  - run pytest on all tests in ./tests
+  - run mypy linting on all files in ./src ./tests
+  - run pre-commit checks:
+
+    - run flake8 style checks against all source
+    - run black formatting checks against all source
+
+While 100% code coverage does not make a library bug-free, it significantly
+reduces the number of easily caught bugs! Please make sure coverage remains the
+same or is improved by a pull request!
+
+Code Styling
+------------
+
+The code in this repository conforms to standards set by the following tools:
+
+- black_ for code formatting
+- flake8_ for style checks
+- isort_ for import ordering
+- mypy_ for static type checking
+
+flake8 and black be run by pre-commit_. You can run the above checks on
```

Comment:
be -> are?

Reply:
You are supposed to read that comment with a west-country accent!

Reply:
done